lib/src/Uncertainty/Algorithm/OrthogonalBasis/AdaptiveStieltjesAlgorithm.cxx

```diff
@@ -177,7 +177,7 @@ Point AdaptiveStieltjesAlgorithm::computeDotProduct(const Function & kernel,
   }
   if (measure_.isDiscrete())
   {
-    const Sample nodes(measure_.getSupport());
+    static const Sample nodes(measure_.getSupport());
     return kernel(nodes).computeMean() * nodes.getSize();
```

Comment:
I am probably missing a point. Could you please recall why using `static` here?

Reply:
It is to avoid a call to getSupport() each time the computeDotProduct() method is called

Reply:
ok thanks. I lost the view of calling the `computeDotProduct ` method many times